examples/apps/face_detection/main.py

```diff
@@ -28,11 +28,11 @@
         movie_name + '_bboxes')
 
     print('Drawing faces onto video...')
-    frame = db.ops.FrameInput()
+    frame = db.sources.FrameColumn()
```

Comment:
I feel like we already had this discussion, but is there a reason to use `sources` and `sinks` instead of `inputs` and `outputs`?

Reply:
My reason is because `sources` and `sinks` is more distinctive than `inputs` and `outputs`. Since inputs and outputs are used so frequently in general, I was predicting that it would be very easy to mix up inputs/outputs with the inputs and outputs of an Op or the inputs/outputs when talking about an algorithm implemented using Scanner.

Thoughts?

Reply:
Ok that's fair. I think people can pick up `source`/`sink` quickly, and I agree that it helps us use a less generic vocabulary.